bots/machine/testvm_core.py

```diff
@@ -0,0 +1,557 @@
+#!/usr/bin/python3 -u
+# -*- coding: utf-8 -*-
+
+# This file is part of Cockpit.
+#
+# Copyright (C) 2013 Red Hat, Inc.
+#
+# Cockpit is free software; you can redistribute it and/or modify it
+# under the terms of the GNU Lesser General Public License as published by
+# the Free Software Foundation; either version 2.1 of the License, or
+# (at your option) any later version.
+#
+# Cockpit is distributed in the hope that it will be useful, but
+# WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+# Lesser General Public License for more details.
+#
+# You should have received a copy of the GNU Lesser General Public License
+# along with Cockpit; If not, see <http://www.gnu.org/licenses/>.
+
+
+import os
+import time
+import socket
+import signal
+import subprocess
+import tempfile
+import select
+import errno
+import sys
+
+BOTS_DIR = os.path.dirname(os.path.dirname(os.path.realpath(__file__)))
+TEST_DIR = os.path.join(os.path.dirname(BOTS_DIR), "test")
+DEFAULT_IDENTITY_FILE = os.path.join(BOTS_DIR, "machine", "identity")
+
+ROOT_USER_TUPLE = ["root", "foobar"]
+ADMIN_USER_TUPLE = ["admin", "foobar"]
+TEST_USER_TUPLE = ["test", "superhardpasswordtest5554"]
```

Comment:
It seems these are not used anywhere. This file also is not specific to our Cockpit VMs (which have the first two at least), I assume the point is to use them for other environments. Also, these are not tuples, but lists :-)

Can you please drop these?

Reply:
yes, I'll drop them, just though that would be nice to have it in some variable instead of hadrcoded on various places, like in classes in bootstrap scripts etc. In next PRs I planned to replace hardcoded values on several places